src/pages/Markets/constants.ts

```diff
@@ -7,3 +11,85 @@ export enum MARKETS_CATEGORIES {
   ONE_CLICK_DEFI = 'oneClickDefiAssets',
   THORCHAIN_DEFI = 'thorchainDefi',
 }
+
+export const sortOptionsByCategory: Record<MARKETS_CATEGORIES, SortOption[] | undefined> = {
+  [MARKETS_CATEGORIES.TRADING_VOLUME]: undefined,
+  [MARKETS_CATEGORIES.MARKET_CAP]: undefined,
+  [MARKETS_CATEGORIES.TRENDING]: [
+    {
+      key: SortOptionsKeys.VOLUME,
+      label: 'dashboard.portfolio.volume',
+    },
+    {
+      key: SortOptionsKeys.PRICE_CHANGE,
+      label: 'dashboard.portfolio.priceChange',
+    },
+    {
+      key: SortOptionsKeys.MARKET_CAP,
+      label: 'dashboard.portfolio.marketCap',
+    },
+  ],
```

Comment:
and then this simplifies to:

```suggestion
  [MARKETS_CATEGORIES.TRENDING]: [SortOptionsKeys.VOLUME, SortOptionsKeys.PRICE_CHANGE, SortOptionsKeys.MARKET_CAP]
```

etc

Reply:
Done in [1ac600a](https://github.com/shapeshift/web/pull/8006/commits/1ac600aeb95da5729ef78865c837fd0bcec0f777)